s3/sc-s3-synapse-ra-v1.0.0.yaml

```diff
@@ -47,43 +54,17 @@ Resources:
             AllowedOrigins: ['*']
             AllowedMethods: [GET, POST, PUT, HEAD]
             MaxAge: 3000
-      Tags:
-        - Key: Name
-          Value: SC-S3-RA-S3-Bucket
-  BucketPolicy:
-    Type: "AWS::S3::BucketPolicy"
-    Condition: AllowUserAccess
+  S3BucketPolicy:
+    Type: Custom::SCS3BucketPolicy
     Properties:
-      Bucket: !Ref 'S3Bucket'
-      PolicyDocument:
-        Version: "2012-10-17"
-        Statement:
-          -
-            Sid: "ReadAccess"
-            Effect: "Allow"
-            Principal:
-              AWS: !Split
-                - ","
-                - !Sub
-                  - "${list},${SynapseProdARN}"
-                  - list: !Join [",",!Ref "S3UserARNs"]
-            Action:
-              - "s3:ListBucket*"
-              - "s3:GetBucketLocation"
-            Resource: !GetAtt S3Bucket.Arn
-          -
-            Sid: "WriteAccess"
-            Effect: "Allow"
-            Principal:
-              AWS: !Split
-                - ","
-                - !Sub
-                  - "${list},${SynapseProdARN}" 
-                  - list: !Join [",",!Ref "S3UserARNs"]
-            Action:
-              - "s3:*Object*"
-              - "s3:*MultipartUpload*"
-            Resource: !Sub "${S3Bucket.Arn}/*"
+      ServiceToken: !ImportValue
+        'Fn::Sub': '${AWS::Region}-cfn-cr-sc-bucket-policy-FunctionArn'
+      BucketName: !Ref S3Bucket
```

Comment:
don't we need to set `RequireEncryption` to true here since BucketEncryption is enabled?

Reply:
The current Synapse bucket does not in fact use the Deny statements. I am reproducing what currently exists in the bucket templates.

Reply:
I still think that it makes more sense to match `BucketEncryption enabled` with `RequireEncryption=true` even though synapse documentation doesn't require it.  I'm guessing it's a bug in the synapse docs.

In any case, i'm not opposed to setting to false either since synapse docs don't mention it currently.